backend/app/services/adapters/executor_job.py

```
@@ -63,129 +80,404 @@ def cleanup_task_executor(
+                total_valid,
+                total_deleted,
+                last_id,
+                total_filter_stats.missing_task,
+                total_filter_stats.invalid_task_payload,
+                total_filter_stats.non_terminal_task,
+                total_filter_stats.stale_non_terminal_task,
+                total_filter_stats.preserve_executor,
+                total_filter_stats.code_task_recent,
+                total_filter_stats.task_updated_recent,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)